hooks/useShowTodayMarker.ts

```diff
@@ -0,0 +1,7 @@
+import { useState } from 'react';
+
+import useSharedHook from '../lib/client/createSharedHook';
+
+const [useShowTodayMarker, setShowTodayMarker] = useSharedHook(useState, false);
+
+export { useShowTodayMarker, setShowTodayMarker };
```

Comment:
I know we talked briefly about hooks vs central state, and I want to say that for this use case I don't imagine anything else will need to call this hook. The `useState` line in `components/roadmap-grid/today-marker-toggle.tsx` is probably the best option, since nothing else should need to use this. lmk if I'm mistaken

Reply:
the grid component in RoadmapDetailedView is using the showTodayMarker state and to avoid passing callbacks to the button/toggle action I took this approach